services/QuillLMS/client/app/bundles/Connect/test/actions/lessons.test.ts

```diff
@@ -6,9 +6,8 @@ jest.mock('../../libs/lessons_api', () => ({
   LessonApi: mockLessonApi,
   TYPE_CONNECT_LESSON: MOCK_LESSON_TYPE,
 }))
-
-import { mockDispatch as dispatch, } from '../__mocks__/dispatch';
-
+// TODO: figure out why these imports need to be ordered here for tests to pass
+import { mockDispatch as dispatch } from '../../../Shared'
```

Comment:
Just out of curiosity, what was the `import` structure that wasn't working? Is it that it has to be below the `jest.mock`?

Reply:
it's quite strange... if I hoisted this to the top, I would get an error that `mockLessonApi` was undefined?